spyder_kernels/console/start.py

```diff
@@ -75,6 +77,53 @@ def sympy_config(mpl_backend):
     return lines
 
 
+def is_anaconda():
+    """
+    Detect if we are running under Anaconda.
+    Taken from https://stackoverflow.com/a/47610844/438386
+    """
+    is_conda = osp.exists(osp.join(sys.prefix, 'conda-meta'))
+    return is_conda
+
+
+def get_conda_root_path():
+    """Get conda root path using 'conda info --json'."""
+    try:
+        conda_info = check_output('conda info --json')
+        conda_info_json = json.loads(conda_info)
+        return conda_info_json['env_vars']['CONDA_ROOT']
+    except Exception:
+        return None
+
+
+def get_conda_env_path():
+    """
+    Get conda environment minimum PATH elements when running under a conda env.
```

Comment:
This should be

    Get minimal set of directories to add to PATH for conda environments on Windows.